doc/audit/node_state.rst

```diff
@@ -0,0 +1,17 @@
+Node State
+==========
+
+.. mermaid::
+
+    graph TB;
```

Comment:
Looking at this again, could we split the graph for each node (start, join, recover)? Also, as this is not recorded in the ledger, should this be moved to the operations section?

Reply:
Very good points!